src/task.c

```diff
@@ -235,6 +235,52 @@ JL_DLLEXPORT void *jl_task_stack_buffer(jl_task_t *task, size_t *size, int *tid)
     return (void *)((char *)task->stkbuf + off);
 }
 
+JL_DLLEXPORT void jl_active_task_stack(jl_task_t *task, char **start, char **end)
+{
+    if (!task->started) {
```

Comment:
We didn't handle this case in `jl_task_stack_buffer`, and I am not sure we need to handle it here? But I thought "better safe than sorry"...